library/src/main/kotlin/display/pathSearchDijkstra/implementation/GraphPathSearchDijkstra.kt

```diff
@@ -0,0 +1,47 @@
+package display.pathSearchDijkstra.implementation
+
+import display.pathSearchDijkstra.GraphPathSearchDijkstra
+import graph.Graph
+
+class GraphDijkstraPathFinder<V, E> {
```

Comment:
то же самое. класс должен реализовать интерфейс

```suggestion
class GraphDijkstraPathFinder<V, E> : GraphPathSearchDijkstra {
```
или (я бы так сделал)
```suggestion
class GraphDijkstraPathFinder<V, E> : GraphPathSearch {
```